### PR TITLE
Update issue templates - bug report

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
Template is generated through repository settings.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+**Environment (please complete the following information):**
+ - **Collection version** (e.g. `helm ls -n sumologic`):
+ - **Kubernetes version** (e.g. `kubectl version`):
+ - **OS** (e.g. from /etc/os-release):
+ - **Kernel** (e.g. `uname -a`):
+ - **Others** (e.g. logs, cloud provider):
+
+**Anything else do we need to know** 
+Add any other context about the problem here, e.g. issue happens only occasionally.
```
Comment (sumo-drosiek) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: Does the final result ( in terms of formatting) is as expected?
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/da244555b15a4eb6773e7dbb65f0512877d84860/.github/ISSUE_TEMPLATE/bug_report.md

Comment (sumo-drosiek) [reply .github/ISSUE_TEMPLATE/bug_report.md:31]: it doesn't matter as I think about that

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. 
+2.
+3.
+4.
```
Comment (sumo-drosiek) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: I would skip that ordered list

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (kkujawa-sumo) [reply .github/ISSUE_TEMPLATE/bug_report.md:31]: I expect that final result will be without the table.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Environment (please complete the following information):**
+ - **Collection version** (e.g. `helm ls -n sumologic`):
+ - **Kubernetes version** (e.g. `kubectl version`):
+ - **OS** (e.g. from /etc/os-release):
+ - **Kernel** (e.g. `uname -a`):
```
Comment (sumo-drosiek) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: Do we need that?

Comment (kkujawa-sumo) [reply .github/ISSUE_TEMPLATE/bug_report.md:27]: If we suppose that the issue may be related to k8s then kernel version may be helpful. Some k8s defects depends on kernel version.

Comment (sumo-drosiek) [reply .github/ISSUE_TEMPLATE/bug_report.md:23]: So is it about `k8s` machines `OS` and `Kernel` or client machine?

Comment (kkujawa-sumo) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I modified the template, I hope it looks better now.

**Review by sumo-drosiek — APPROVED** [review]

**Pull request merged**